generate_script/index.js

```diff
@@ -40,76 +38,88 @@ function generateCertificate(keys) {
   const attrs = [
     {
       name: "commonName",
-      value: "example.org",
-    },
-    {
-      name: "countryName",
-      value: "US",
-    },
-    {
-      shortName: "ST",
-      value: "California",
-    },
-    {
-      name: "localityName",
-      value: "San Francisco",
-    },
-    {
-      name: "organizationName",
-      value: "Example Organization",
-    },
-    {
-      shortName: "OU",
-      value: "Test",
-    },
+      value: hub_domain,
+    }
   ];
   cert.setSubject(attrs);
   cert.setIssuer(attrs);
 
   // Sign the certificate with its own private key
-  cert.sign(forge.pki.privateKeyFromPem(privateKey), forge.md.sha256.create());
+  cert.sign(privateKey, forge.md.sha256.create());
 
-  // Convert the certificate to PEM format
+  // Convert the certificate and private key to PEM format
   const pemCert = forge.pki.certificateToPem(cert);
+  const pemPrivateKey = forge.pki.privateKeyToPem(privateKey);
 
-  return { pemCert, privateKey, publicKey };
+  return { pemCert, pemPrivateKey };
 }
 
 // Function to convert PEM to JWK
-async function convertPemToJwk(publicKey) {
+function convertPemToJwk(publicKey) {
   const jwk = pemJwk.pem2jwk(publicKey);
   return JSON.stringify(jwk);
 }
 
+function generatePersistentVolumes(processedConfig, replacedContent) {
+  if (processedConfig.KUBERNETES_PROVIDER === "DO") {
+    processedConfig.ACCESS_MODE = "ReadWriteMany";
```

Comment:
Digital Ocean also uses ReadWriteOnce

Reply:
https://docs.digitalocean.com/products/kubernetes/how-to/add-volumes/

Reply:
Ah, okay.  I thought from your previous PR on the Hubs Cloud repo that ReadWriteMany was the way to go for DigitalOcean, but since that's not the case, I'll switch it to ReadWriteOnce.  Are there any platform differences now or can remove this whole kubernetes provider thing?